mozilla_aws_cli/listener.py

```diff
@@ -113,17 +117,17 @@ def get_heartbeat():
             "status_code": 500,
         })
 
-    start = time.time()
-    while time.time() - start < 30:
-        if login.last_state_check is None:
-            pass
-        elif (time.time() - login.last_state_check >
-              login.max_sleep_no_state_check):
-            logger.error(
-                "No response from web interface for {} seconds, shutting "
-                "down.".format(login.max_sleep_no_state_check))
-            exit_sigint()
-        time.sleep(0.5)
+    # start = time.time()
+    # while time.time() - start < 30:
+    #     if login.last_state_check is None:
+    #         pass
+    #     elif (time.time() - login.last_state_check >
+    #           login.max_sleep_no_state_check):
+    #         logger.error(
+    #             "No response from web interface for {} seconds, shutting "
+    #             "down.".format(login.max_sleep_no_state_check))
+    #         exit_sigint()
+    #     time.sleep(0.5)
```

Comment:
I will require guidance here:  This 2 second response time is too strict on my machine.

Reply:
...plus it seems to delay response for 30 seconds